lib/server.js

```diff
@@ -353,6 +353,15 @@ module.exports = class Server extends EventEmitter {
 
   _createServer() {
     this._httpServer = this._createHttpServer();
+
+    if (this.opts.http2 || this.opts.secure) {
+      let port = 80 // TODO : this should be this.opts.httpPort or this.opts.insecurePort & 80 by default if none provided
```

Comment:
This comment addresses good points, those options should exist.

Reply:
@ericmdantas what should we use here `httpPort` , `insecurePort` , or something else ?